superset-frontend/images/icons/check.svg

```diff
@@ -0,0 +1,22 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+ 
+    http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M12 2C6.47715 2 2 6.47715 2 12C2 17.5228 6.47715 22 12 22C17.5228 22 22 17.5228 22 12C22 9.34784 20.9464 6.8043 19.0711 4.92893C17.1957 3.05357 14.6522 2 12 2Z" fill="#666666"/>
```

Comment:
One more comment, I think you might need to default the `fill` attribute to `"currentColor"` so that we can override the color of the icon with CSS

Reply:
[CSS rules end up overriding the property.](https://css-tricks.com/almanac/properties/f/fill/) However, it's best to be consistent and use `currentColor`.

Reply:
ah, I see. I was using `color` to change the icon's color and not `fill` so it didn't work. Do we prefer to override this with `color` or `fill` on the icon? i was planning a change to the Icon component to get rid of `styled` and use this method instead; with this then `color` works:
```
const Icon = ({ name, color = '#666666', ...rest }: IconProps) => {
  const Component = iconsRegistry[name];
  return <Component color={color} {...rest} />;
};

export default Icon;
```

Reply:
color seems reasonable, as that's how the icon fonts behave. We should def try to use `currentColor` for fill rules when it makes sense as it allows for the most flexibility.